### PR TITLE
No tags message in account screen

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#304** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/304?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/screen.tsx [hunk 1]
```diff
@@ -237,3 +243,51 @@ const ProfileFacts = ({ facts }: { facts: { label: string; value?: React.ReactNo
...
+          </Paragraph>
+          <Paragraph fontSize={'$4'} fontWeight={'500'} color={'$color12'} fontFamily={'$mono'}>
+            2. Send and receive funds using your personalized, easy-to-remember Sendtag
+          </Paragraph>
+          <Paragraph fontSize={'$4'} fontWeight={'500'} color={'$color12'} fontFamily={'$mono'}>
+            3. Claim your preferred Sendtag before someone else does
+          </Paragraph>
+        </YStack>
+        <Paragraph fontSize={'$4'} fontWeight={'500'} color={'$color12'} fontFamily={'$mono'}>
+          Join 5,676 Senders in shaping the Future of Finance.
```
Comment (0xBigBoss) [inline packages/app/features/account/screen.tsx:270]: let's drop this content as the number is inaccurate, not worth keeping in.

**Review by 0xBigBoss — APPROVED** [review]
LGTM but can we also add some bottom padding. The buttons are right up against the edge of the container.

sendapp-no-tags-message-in-account-screen-0xsend vercel app_account(iPad Air)

sendapp-no-tags-message-in-account-screen-0xsend vercel app_account(iPhone SE)

File: packages/app/features/account/screen.tsx (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/features/account/screen.tsx:265]: O my bad, I can just pull a user count in for this

**Pull request merged**